plugins/importexport/native/pkp-native.xsd

```diff
@@ -187,6 +187,14 @@
 			<element name="givenname" type="pkp:localizedNode" minOccurs="1" maxOccurs="unbounded" />
 			<element name="familyname" type="pkp:localizedNode" minOccurs="0" maxOccurs="unbounded" />
 			<element name="affiliation" type="pkp:localizedNode" minOccurs="0" maxOccurs="unbounded" />
+            <element name="rorAffiliation" minOccurs="0" maxOccurs="unbounded">
+                <complexType>
+                    <sequence>
+                        <element name="ror" type="string" minOccurs="0" maxOccurs="unbounded" />
+                        <element name="name" type="pkp:localizedNode" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
Should the minOccurs for ror and name be 1 -- if there is a rorAffiliation, then both elements should exist, no?